doc/source/ray-job-submission/overview.rst

```diff
@@ -0,0 +1,239 @@
+
+
+=========================
+From laptop to production
+=========================
+
+.. warning::
+
+    Ray Job Submission is at Alpha phase with APIs mostly stable but subject to change in the future.
+
+Concepts
+--------
+
+- **Package**: A collection of files and configurations that defines an application, thus allowing it to be executed in a different environment remotely (ideally self-contained).
+
+- **Job**: A Ray application that will be submitted to a Ray cluster for execution. Once a job is submitted, it runs once on the cluster to completion or failure. Retries or different runs with different parameters should be handled by the submitter. Jobs are scoped to the lifetime of a ray cluster.
+
+- **Job Manager**: An entity that manages the lifecycle of a Job and potentially multiple ray clusters, such as scheduling, killing, polling status, getting logs, and persisting inputs / outputs. Should be highly available (HA) by default. Can be any framework with these abilities, such as Airflow.
```

Comment:
We should be consistent about capitalizing Job, but not sure whether uppercase or lowercase is better.

Reply:
Let's also capitalize Ray everywhere